CHANGES.md

```diff
@@ -2,9 +2,8 @@
 
 ## Removed
 
- - Remove `iter` and `force_merge` from `Index.S`, due to difficulties with
-   guaranteeing sensible semantics for these functions under MRSW access
-   patterns. (#147)
+ - Remove `force_merge` from `Index.S`, due to difficulties with guaranteeing
```

Comment:
The difficulties mentioned here are related to `iter`, no?

Reply:
They're still related to `force_merge`, I think. If we need to support `iter`, then I suppose the best we can do is give a big warning in the documentation comment as @pascutto suggests.